setup.ts

```diff
@@ -1015,25 +1015,36 @@ async function main(): Promise<void> {
     const REDIS_PASSWORD = "";
     const MINIO_ENDPOINT = "http://minio:9000";
 
+    const { pwdVariable } = await inquirer.prompt({
+      type: "input",
+      name: "pwdVariable",
+      message:
```

Comment:
Why was this added? It should have been fixed in a previously merged PR. Please merge your code with the latest `upstream/postgres-develop` to be certain.

Reply:
sir my branch is working against develop branch, not `develop-postgres.`.merging `develop-postgres` would result in a lot of conflicts, I tried to pull from develop but I am already up to date with the `upstream-develop` branch